consensus/raft/raft.go

```diff
@@ -302,13 +256,13 @@ func (r *Raft) RemovePeer(peer string) error {
 }
 
 // func (r *Raft) SetPeers(peers []string) error {
-// 	logger.Debugf("SetPeers(): %s", peers)
```

Comment:
Should SetPeers be removed?

Reply:
I'm not sure, probably when I deal with the new Raft version.